ui/lib/core/addon/components/form-field.hbs

```diff
@@ -40,7 +40,12 @@
                 class="has-left-margin-xs has-text-black is-size-7"
                 data-test-radio-label={{or val.label val.value val}}
               >
-                <span>{{or val.label val.value val}}</span>
+                {{or val.label val.value val}}
```

Comment:
so this or leaves the value if there is a val.label or val.value? I realize you are just removing the span here, but this just reads a little odd to me.

Reply:
`@attr.options.possibleValues` can be an array of strings or an array of objects. Sometimes the value _is_  what we want to display, but other times we want the label to be something different than the backend param. So this is saying, is there a label otherwise just use the value itself. I think the triple `or` is a little odd, though!